bigchaindb/backend/mongodb/connection.py

```diff
@@ -60,6 +64,10 @@ def _connect(self):
                     time.sleep(2**i)
 
 
+def table(name):
```

Comment:
Might be more canonical to name this `collection`, given that that's what mongo names them (and this is a mongo specific implementation).